packages/mjml-text/README.md

```diff
@@ -1,6 +1,15 @@
 ## mjml-text
 
-This tag allows you to display text in your email.
+Most important, this element places text in your email.
+
+Also important, this element supports placement of raw HTML tags
+  and their attributes anywhere within.
+Sometimes, developers put _only_ HTML within a given `<mj-text>` tag set.
+
+Inside this tag, there's no need to surround each paragraph with HTML `<p>` tags.
+The HTML `<br/>` tag is convenient for creating  line breaks.
+
+
```

Comment:
An email seemed to indicate you wanted me to remove, "Don't forget to encode special characters to avoid unexpected behaviour from MJML's parser". ("This was an issue in MJML3, and it's no longer the case so we can remove this phrase").

I removed it. I mention it because since your note isn't reproduced here, I could have misinterpreted.